server/components/form-settings/form-settings.html

```diff
@@ -150,37 +150,47 @@
     accept: 'image/*'
   }) }}
 
-  {% call radioWrapper({
-    label: "Lastly, what is your coding level?",
-    hook: 'level-form',
+  {{ input({
+    name: 'song',
+    id: 'song',
+    label: 'One more step, what is your favorite song?',
+    placeholder: 'The A Team',
+    hook: 'input-song'
+  }) }}
+
+  {{ input({
+    name: 'artist',
+    id: 'artist',
+    label: 'By which artist?',
+    placeholder: 'Ed Sheeran',
+    hook: 'input-artist'
+  }) }}
+
+  {% call select({
+    label: 'Which genre does it belong to?',
+    hook: 'genre-form',
     class: 'form__item--is-visible'
   }) %}
 
-    {{ radio({
-      name: 'level',
-      value: 'Noob',
-      image: '/assets/images/noob.png'
-    }) }}
+  <option name='genre' value='deephouse'>Deephouse</option>
```

Comment:
Maybe we can write a loop over all genres

Reply:
Good idea, we should do this! Also create a option form element macro